src/osd/OSD.cc

```diff
@@ -7999,6 +8018,40 @@ void OSD::trim_maps(epoch_t oldest, bool skip_maps)
   ceph_assert(min <= service.map_cache.cached_key_lower_bound());
 }
 
+int OSD::trim_stale_maps(epoch_t& last_epoch)
```

Comment:
I'd prefer changing the in/out param into accepting the epoch by value, and returning two values (best: in a named structure)

Reply:
last_epoch was removed since the change to a single call to trim_stale_maps